src/sentry/plugins/sentry_mail/activity/new_processing_issues.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import
+
+from sentry.models import User, GroupSubscriptionReason, EventError
+from sentry.utils.http import absolute_uri
+
+from .base import ActivityEmail
+
+
+def summarize_issues(issues):
+    rv = []
+    for issue in issues:
+        extra_info = None
+        msg_d = dict(issue['data'])
+        msg_d['type'] = issue['type']
+
+        if 'image_path' in issue['data']:
+            extra_info = issue['data']['image_path'].rsplit('/', 1)[-1]
+            if 'image_arch' in issue['data']:
+                extra_info = '%s (%s)' % (
+                    extra_info,
+                    issue['data']['image_arch'],
+                )
+
+        rv.append({
+            'message': EventError.get_message(msg_d),
+            'extra_info': extra_info,
+        })
+    return rv
+
+
+class NewProcessingIssuesActivityEmail(ActivityEmail):
+
+    def __init__(self, activity):
+        ActivityEmail.__init__(self, activity)
+        self.organization = self.project.organization
+        self.issues = summarize_issues(self.data['issues'])
+
+    def get_participants(self):
+        # XXX: We want to send an email to everybody who is subscribed to
+        # the mail alerts.  Since currenlty that is only checked in the
+        # base notify code and requires event information, we use the UI
+        # code (Project.is_user_subscribed_to_mail_alerts) which
+        # replicates the logic on a per-user basis.
+        users = User.objects.filter(
+            sentry_orgmember_set__teams=self.project.team,
+            is_active=True,
+        )
+        participants = {}
+        for user in users:
+            if self.project.is_user_subscribed_to_mail_alerts(user):
```

Comment:
This is less than ideal, I agree. Doing this is potentially two queries per user. And there many be hundreds of members on the org. I'd like to see `is_user_subscribed_to_mail_alerts` either be refactored to accept a list of users, or add another method for checking this in bulk. Since in theory, this can be constant time with only 2 queries no matter how many users there are.